.travis.yml

```diff
@@ -1,7 +1,8 @@
 language: python
 python:
-  - "2.6"
-  - "2.7"
+  - 2.6
```

Comment:
You'll need to update the `exclude` configuration block below to remove the quotes around `"2.6"` to get the Travis CI builds to pass.